lib/utils/networks.js

```diff
@@ -165,6 +185,7 @@ export const formatBlockExplorerTxUrl = (tx, networkId) => {
       case 42:
```

Comment:
I guess the block explorer URLs are pretty standardized, so we could remove this big switch